edx_prefectutils/snowflake.py

```diff
@@ -446,7 +446,8 @@ def export_snowflake_table_to_s3(
               field values only. Defaults to snowflake default `\\`.
       null_marker (str, optional): String used to convert SQL NULL. Defaults to `NULL`.
       overwrite (bool, optional): Whether to overwrite existing data in S3. Defaults to `TRUE`.
-      single (bool, optional): Whether to generate a single file in S3. Defaults to `TRUE`.
+      single (bool, optional): Whether to generate a single file in S3. Defaults to `FALSE`. The default
```

Comment:
I think remove the word `default` before `max` is better. The word `default max` is super confusing to me. What if max is exceeded, what happens to this job?

Reply:
If the max file size is exceeded, you'll get an error like 
`snowflake.connector.errors.ProgrammingError: 100083 (22000): Max file size (16777216) exceeded for unload single file mode. We recommend that you increase the max file size parameter, or disable single-file mode in the unload command and combine the unloaded files into a single file after you download them.`

The reason I added `default` is because the max file size can be added as an option to this copy command as well, and the default value for that option is 16MB. I can update the comment to better reflect that.